tests/apichecks/testdata/exceptions/missingfields.txt

```diff
@@ -18,6 +18,9 @@
 [missing_field] crd=accesscontextmanageraccesslevels.accesscontextmanager.cnrm.cloud.google.com version=v1beta1: field ".spec.custom.expr.expression" is not set in unstructured objects
 [missing_field] crd=accesscontextmanageraccesslevels.accesscontextmanager.cnrm.cloud.google.com version=v1beta1: field ".spec.custom.expr.location" is not set in unstructured objects
 [missing_field] crd=accesscontextmanageraccesslevels.accesscontextmanager.cnrm.cloud.google.com version=v1beta1: field ".spec.custom.expr.title" is not set in unstructured objects
+[missing_field] crd=accesscontextmanageraccesslevels.accesscontextmanager.cnrm.cloud.google.com version=v1beta1: field ".spec.description" is not set in unstructured objects
+[missing_field] crd=accesscontextmanageraccesspolicies.accesscontextmanager.cnrm.cloud.google.com version=v1beta1: field ".spec.title" is not set in unstructured objects
```

Comment:
I was spot checking this one. It looks like we don't have any test for this beta resource?

Reply:
Good catch! Yeah, we didn't test it because it requires a new specific organization for testing. We should support it whenever we have the capacity to set the test environment up!